src/pages/CreatePartyPage.jsx

```diff
@@ -55,24 +82,19 @@ const CreatePartyPage = () => {
 
   const handleNext = () => {
     setActiveStep(prevActiveStep => prevActiveStep + 1);
-    if (activeStep === 0 || activeStep === 1 || newParty.mustFilled !== null) {
+    if (activeStep === 0 || newParty.mustFilled !== null) {
```

Comment:
```suggestion
    const 적당한 변수명 = (activeStep  && newParty.mustFilled === null)
    if (적당한변수명) {
      setStepComplete(true)
    }
```
early return을 하는 것도 좋지만 이후 로직이 복잡하지는 않아서 if문의 조건을 긍정조건으로 변환하고 좀 더 명확한 식별자를 사용하면 가독성이 좋을 것 같아요!